lib/u3d/commands.rb

```diff
@@ -342,6 +341,45 @@ def enforce_setup_coherence(packages, options, unity, definition)
         true
       end
 
+      def skippable_packages(unity, packages)
+        return [] if unity.packages.nil?
+        result = []
+
+        installed_packages = unity.packages
+        packages.each do |pack|
+          result << pack if package_installed?(pack, installed_packages)
+        end
+
+        result
+      end
+
+      def package_installed?(package, installed_packages)
```

Comment:
would it be better to move this to unity? I don't think command should have too much logic anyway.

E.g. if `unity.package_installed?` existed, it would be less code here.